### DISCUSSION TITLE
Can the Zappi's scheduled boost be desplayed in HA?

### DISCUSSION THREAD

**Discussion by chaz2000** [CJNE/ha-myenergi#448] [Ideas]
I would like to build a simple lamp that shines different colours depending on the state of the Zappi. The aim would be to give me and the wife a quick visual reminder (before going up to bed). Ideally, I would like it to display if there is a boost scheduled in the next 24 hours. i feel that this is doable with an HA automation but the schedule info is not currently available.

**Comment by G6EJD** [comment]
The API doesn’t contain schedule information, it’s a write-only function to the device.
So you determine the schedule and it follows, so no need to read the API as you’d already have the schedule!

**Reply by LeiChat** [reply]
Or we just have not discovered the endpoint for it yet?
If my wife or I modify a zappi scheduled boost in the myenergi app on either of our phones the change is synchronised in the other.

**Comment by chaz2000** [comment]
Thanks for letting me know. I think should be able to come up with something else.

**Comment by CosAdrPala** [comment]
Hello, Any update on this point?

**Comment by G6EJD** [comment]
You'd have to monitor the MyEnergi API responses to discover if there was any schedule information present, and currently the consensus is there's none.  
This is a community generated API interface, so unless someone does the research, this is never going to happen.
Schedule updates are only stored on the MyEnergi servers, not on the Zappi or other devices.

**Comment by LeiChat** [comment]
I have had a search on the official myenergi forums and the closest I can find to a request is this thread https://myenergi.info/viewtopic.php?p=10390
